tests/test_mssqlcliclient.py

```diff
@@ -107,32 +108,42 @@ def test_json_writer_extra_params(self):
             json_writer.close()
             shutdown(client)
 
-    def test_schema_table_views_and_columns_query(self):
+    @staticmethod
+    def test_schema_table_views_and_columns_query():
```

Comment:
We've had this same issue elsewhere in tools. The standard solutions are:
- Spin up a new DB for each test run & dispose after, and/or:
- Run the drop statements both before and after the creates. This means that if they were left over from previous run you can remove. Otherwise you end up with a lot of crufty tables lying around when someone hits Stop halfway through a test.

Have you considered these options?

Reply:
Thanks @kevcunnane. We've considered the first option but ran into more problems with creating more databases, including failures from lengthy test times. But I like the second option a lot, I'll add that and push out another commit.